integration_tests/run_e2e_tests.py

```diff
@@ -376,21 +413,30 @@ def main(target, e2e_type, generate_data, clear_tests):
         e2e_targets = [target]
 
     if e2e_type == 'all':
-        e2e_types = ['table', 'column', 'schema', 'regular', 'artifacts', 'error_test', 'error_model', 'error_snapshot', 'dimension']
+        e2e_types = ['table', 'column', 'schema', 'regular', 'artifacts', 'error_test', 'error_model', 'error_snapshot',
+                     'dimension']
     else:
         e2e_types = [e2e_type]
 
     all_results = {}
+    found_failures = False
     for e2e_target in e2e_targets:
         print(f'Starting {e2e_target} tests\n')
         e2e_test_results = e2e_tests(e2e_target, e2e_types, clear_tests)
         print(f'\n{e2e_target} results')
         all_results[e2e_target] = e2e_test_results
 
     for e2e_target, e2e_test_results in all_results.items():
-        print(f'\n{e2e_target} results')
-        print_tests_results(*e2e_test_results)
+        failed_test_results = [test_result for test_result in e2e_test_results if not test_result.success]
+        print(f'\n[{len(e2e_test_results) - len(failed_test_results)}/{len(e2e_test_results)}] TESTS PASSED')
```

Comment:
1. Could you please print it after printing the failures (as this is the bottom line)?
2. Could you print also the e2e target?